parsers/test/test_BG.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime, timedelta, timezone
+
+import pytest
+import requests_mock
+from freezegun import freeze_time
+
+from electricitymap.contrib.parsers.lib.exceptions import ParserException
+from parsers.BG import SOURCE_API_URL, fetch_production
+
+
+@pytest.fixture()
+def fixture_requests_mock():
+    """A mocker for requests.
+
+    TODO(amv213): replace with builtin pytest 'requests_mock' fixture when upgrading to requests-mock>=1.5.0
```

Comment:
Thanks for the link @VIKTORVAV99 - I will use that template / switch to `unittest`-style tests

thought: maybe it would be less awkward on `requests-mock>=1.5.0` where `requests_mock` is available as a builtin fixture, but this is what that entire `test_CEB.py` file would look like `pytest`-style: 
```python
import json
from importlib import resources

from requests_mock import ANY

from electricitymap.contrib.lib.types import ZoneKey
from parsers.CEB import fetch_production


def test_production(snapshot, requests_mock):
    requests_mock.get(
        ANY, 
        json=json.loads(
            resources.files("parsers.test.mocks.CEB")
            .joinpath("response.text")
            .read_text()
        ),
    )

    production = fetch_production(zone_key=ZoneKey("LK"))
    
    snapshot.assert_match(
        [
            {
                "datetime": element["datetime"].isoformat(),
                "production": element["production"],
                "storage": element["storage"],
                "source": element["source"],
                "zoneKey": element["zoneKey"],
                "sourceType": element["sourceType"].value,
            }
            for element in production
        ]
    )

```
which I find more readable / maintainable... but I'm probably just biased because I haven't written `unittest`-style tests in a very long time 😉 

Either way, I completely agree with your suggestion! It will also avoid having the test structure / framework diverge across parsers which is probably the most important thing

Reply:
We might be able to upgrade the package and streamline the tests even further at one point but I don't think that should be a goal in this PR.